insta/src/lib.rs

```diff
@@ -170,27 +170,28 @@
 //!
 //! The following features exist:
 //!
-//! * `csv`: enables CSV support (via serde)
-//! * `json`: enables JSON support (via serde)
-//! * `ron`: enables RON support (via serde)
-//! * `toml`: enables TOML support (via serde)
-//! * `yaml`: enables YAML support (via serde)
+//! * `csv`: enables CSV support (via [`serde`])
+//! * `json`: enables `JSON` support (via [`serde`])
+//! * `ron`: enables RON support (via [`serde`])
+//! * `toml`: enables TOML support (via [`serde`])
+//! * `yaml`: enables YAML support (via [`serde`])
 //! * `redactions`: enables support for redactions
 //! * `filters`: enables support for filters
-//! * `glob`: enables support for globbing ([`glob!`])
+//! * `glob`: enables support for globbing ([`crate::glob!`])
```

Comment:
Did the existing version work OK? If so, the proposed version seems over-specified

Reply:
Interestingly, what does not work in https://github.com/mitsuhiko/insta/pull/618/files#r1777903673 does work here.. 🤯

I don't understand rustdoc's scoping today..